appveyor.yml

```diff
@@ -9,10 +9,6 @@ branches:
     - develop
     - sarif-v1
 
-before_build:
-  - '"%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Community\Common7\Tools\VsDevCmd.bat"'
```

Comment:
>VsDevCmd [](start = 79, length = 8)

No need for this; AppVeyor automatically sets up a VS command prompt window for you (as it turns out).